test-clients/src/main/java/com/hedera/services/bdd/suites/SuiteRunner.java

```diff
@@ -658,12 +658,22 @@ static private String rightPadded(String s, int width) {
 		}
 	}
 
-	@SafeVarargs
-	public static <T> Supplier<T[]> aof(Supplier<T>... items) {
-		return () -> (T[]) List.of(items)
```

Comment:
I think the problem is `(T[])` casting is not type-safe. You should also delete the old code, @ljianghedera.

Reply:
It seems like so. I googled around, but no particular explanation why. The old code seems to be fine.  
(I kept the old code as a reminder for us to revisit it later when we have spare time.)